app/views/tweeets/_feed.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="column is-half">
+  <% if user_signed_in? %>
+    <article class="media box">
+      <figure class="media-left">
+        <p class="image is-64x64">
+          <%= gravatar_image_tag(current_user.try(:email), size: 64, alt: current_user.try(:name)) %>            
+        </p>
+      </figure>
+      <div class="media-content">
+        <%= render "tweeets/form"%>
+      </div>
+    </article>
+  <%end%>
+  <% @tweeets.each do |tweeet| %>
+    <div class="box">
+      <article class="media">
+        <div class="media-left">
+          <figure class="image is-64x64">
+            <%= gravatar_image_tag(current_user.try(:email), size: 64, alt: current_user.try(:name)) %>                
+          </figure>
```

Comment:
**REQUIRED**
* Let's move all the views related logic to the helpers.
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)
* [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright

Reply:
done